tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -868,7 +868,6 @@ Then('the app-sidebar should be invisible', async function () {
 })
 
 Then('the {string} details panel should be visible', async function (panel) {
-  await client.page.FilesPageElement.appSideBar().activatePanel(panel)
```

Comment:
We should not perform any actions in `Then` step. We must only check if the panel is visible or not